src/coreclr/vm/interpexec.cpp

```diff
@@ -129,6 +131,63 @@ void InvokeDelegateInvokeMethod(MethodDesc *pMDDelegateInvoke, int8_t *pArgs, in
     pHeader->Invoke(pHeader->Routines, pArgs, pRet, pHeader->TotalStackSize);
 }
 
+void InvokeUnmanagedCalliWithTransition(PCODE ftn, void *cookie, int8_t *stack, InterpMethodContextFrame *pFrame, int8_t *pArgs, int8_t *pRet)
```

Comment:
This should be marked with `NOINLINE`, and it can be moved next to InvokeUnmanagedMethodWithTransition outside WASM ifdef.